src/utility/Math.cpp

```diff
@@ -0,0 +1,47 @@
+#include "utility/Math.h"
+
+#ifndef PINETIME_IS_RECOVERY
+
+  #include <lvgl/src/lv_misc/lv_math.h>
+
+int16_t Asin(int16_t arg) {
```

Comment:
Does it work, if you add
```cpp
if (PINETIME_IS_RECOVERY) {
  return 0;
}
```
here instead and let the compiler figure out that the include is not needed?

Reply:
No, because `PINETIME_IS_RECOVERY` isn't defined at all if it's not building the recovery, so that'll just give an error.